src/hadoop-data-node/deploy/start.sh

```diff
@@ -28,14 +28,14 @@ pushd $(dirname "$0") > /dev/null
 # Hadoop data node
 kubectl apply --overwrite=true -f hadoop-data-node.yaml || exit $?
 
-PYTHONPATH="../.." python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k hadoop-data-node -v "true"
+PYTHONPATH="../../../deployment" python -m  k8sPaiLibrary.monitorTool.check_node_label_exist -k hadoop-data-node -v "true"
```

Comment:
@YitongFeng  To ensure, whether this line could be removed or not.

Reply:
could removed